src/sessions/SessionsSection.tsx

```diff
@@ -51,7 +45,7 @@ export default function SessionsSection() {
 
     setNoSessions(clientSessions.length === 0);
 
-    const allClients = await adminClient.clients.find();
+    const allClients = await adminClient.clients.find({ realm });
```

Comment:
Realm here is not needed as the adminClient is already setup with the active realm
```suggestion
    const allClients = await adminClient.clients.find();
```